mewarpx/mewarpx/runinfo.py

```diff
@@ -188,10 +190,12 @@ def __init__(self, injector_dict, surface_dict,
         self._reserved_names = (["cathode_all", "anode_all",
                                  "aside", "aside_nodiel"] +
                                 [x + "_all" for x in self.component_list])
-        self._check_voltages_wfs(check_names=True)
-
-        # Split out saving other stuff into functions
-        self._save_sim_info()
+        callbacks.installafterinit(self._check_voltages_wfs)
```

Comment:
Just in case you are not aware of this, you can do
```
        callbacks.installafterinit(partial(self._check_voltages_wfs, True))
```
to execute `self._check_voltages_wfs(True)` after init.